src/components/web/Button.jsx

```
@@ -1,6 +1,14 @@
 import PropTypes from 'prop-types'
 
-const Button = ({ label, size = 'medium', onClick, bgColor = 'light-gray', className = '' }) => {
+const Button = ({
+  label,
+  size = 'medium',
+  onClick,
+  disabled = false,
```

ESLint

[error] 'disabled' is missing in props validation

(react/prop-types)